corehq/motech/openmrs/atom_feed.py

```diff
@@ -146,7 +151,9 @@ def has_new_entries_since(last_polled_at, element, xpath='./atom:updated'):
             if has_new_entries_since(last_polled_at, feed_xml):
                 for entry in feed_xml.xpath('./atom:entry', namespaces={'atom': 'http://www.w3.org/2005/Atom'}):
                     if has_new_entries_since(last_polled_at, entry, './atom:published'):
-                        yield get_uuid(entry)
+                        entry_uuid = get_uuid(entry)
+                        if entry_uuid:
+                            yield entry_uuid
```

Comment:
@millerdev, polling for your opinion: Do you these three lines should be written as ...
```python
    if entry_uuid := get_uuid(entry):
        yield entry_uuid
```
... in Python 3.7?

Reply:
lol! I probably would not do that. I think my criteria for using `:=` will be if it removes two or more lines without making the resulting code harder to read. So this seems reasonable:
```diff
- while True:
-     thing = get_thing()
-     if thing:
-         break
+ while thing := get_thing():
      do(thing)
```
On the other hand, [Tim Peters likes that pattern](https://www.python.org/dev/peps/pep-0572/#appendix-a-tim-peters-s-findings) (search for "Name binding is very frequent"), so there are certainly respectable developers out there who would do it.

Also, `:=` will be available in Python 3.8, not 3.7